shell/common/shell.cc

```diff
@@ -713,6 +713,9 @@ bool Shell::Setup(std::unique_ptr<PlatformView> platform_view,
   weak_rasterizer_ = rasterizer_->GetWeakPtr();
   weak_platform_view_ = platform_view_->GetWeakPtr();
 
+  if (settings_.enable_implicit_view) {
+    engine_->AddView(kFlutterImplicitViewId, ViewportMetrics{});
```

Comment:
From my understanding, at this point the Dart isolate isn't ready yet to receive the view? If yes, could you add a quick comment that this view won't actually be added until `RuntimeController::FlushRuntimeStateToIsolate`.

Reply:
The Dart isolate isn't ready to receive this view for sure, as well as all other views that are added immediately after `FlutterEngine` is constructed. These view operations are stored in `RuntimeController` and not sent to Dart until `FlushRuntimeStateToIsolate`. I can note this in `FlushRuntimeStateToIsolate`.